.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
Yeah, and that what rollup-plugin-babel do. External plugin just add namespace to helpers which is used by rollup-plugin-babel. But that happen without additional and not always efficient work by commonjs plugin and expliit dependency which developer should specify instead of what exactly plugins require.